modules/va_facilities/spec/requests/facilities_request_spec.rb

```diff
@@ -347,5 +347,16 @@ def query_params(url)
       get base_query_path, params: 'state=meow', headers: accept_json
       expect(response).to have_http_status(:bad_request)
     end
+    it 'returns 400 for more than one distance location param type' do
+      get base_query_path + pdx_bbox + '&state=FL' + '&type=benefits&services[]=DisabilityClaimAssistance'
+
+      json = JSON.parse(response.body)
+      expect(response.status).to eq(422)
+      expect(json['errors'].first).to eq(
+        'You may only use ONE of these distance query parameter sets: lat/long, zip, state, or bbox'
+      )
+
+      expect(response).to have_http_status(:unprocessable_entity)
```

Comment:
I don't think you need both 
```
expect(response.status).to eq(422)
```
and
```
expect(response).to have_http_status(:unprocessable_entity)
```
as one is checking the status code itself and the other is checking the symbol of that status code.

More generally, when writing error response specs I've just been using `have_http_status` to test. Just wondering if there is a consensus on parsing the response so that the error message can be checked as well? Asking now before I get to code review. :smile:

Reply:
That's a good point. It seems like the other tests in this suite only use `have_http_status` so I _think_ that's the convention.